src/models/User.model.js

```diff
@@ -1,9 +1,26 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const User = sequelize.define(
-  // your code goes here
+  'User',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'users',
+    updatedAt: false,
+    createdAt: false,
```

Comment:
Consider enabling `timestamps` by setting `timestamps: true` in the model options. This will automatically add `createdAt` and `updatedAt` fields to the model, which can be useful for tracking changes.